app/routes/news._index.mdx

```diff
@@ -26,6 +26,42 @@ import { Anchor } from '~/components/Anchor'
 
 ## 2024
 
+    <CollectionItem
+      className="maxw-none"
+      variantComponent={
+        <CollectionCalendarDate datetime={'May 20, 2024'} />
+      }
+    >
+      <CollectionHeading headingLevel="h3">
+        <Anchor>New Einstein Probe Notices, Astro Flavored Markdown for Circulars, and Notices Schema v4.0.0</Anchor>
+      </CollectionHeading>
+      <CollectionDescription>
+
```

Comment:
Add a short introductory sentence here... something to the effect of "Lots of new features this month!"